2.4/examples/replica/mongodb-clustered.json

```diff
@@ -119,6 +119,20 @@
               {
                 "name":  "mongodb",
                 "image": "openshift/mongodb-24-centos7",
+                "readinessProbe": {
+                  "timeoutSeconds": 1,
+                  "initialDelaySeconds": 3,
+                  "exec": {
+                    "command": [ "/bin/sh", "-i", "-c", "mongostat --host 127.0.0.1 -u admin -p $MONGODB_ADMIN_PASSWORD -n 1 --noheaders"]
```

Comment:
@mfojtik It depends on the purpose of this call. To simply detect that MongoDB accepts connection - `mongo --eval "quit()"`, in case of replication it is complicated.